resources/privacy-and-gdpr.md

```diff
@@ -228,15 +229,17 @@ The following API returns a dictionary representation of the consent preferences
 
 ## Retrieving stored identifiers
 
-To retrieve all the identifier data stored locally by the SDK as a JSON string, and send this data to your servers, use the [getSdkIdentities](../using-mobile-extensions/mobile-core/mobile-core-api-reference.md#getsdkidentities) API from the Mobile Core extension.
+When using the Edge Network extensions, use the [Identity.getIdentities](../using-mobile-extensions/adobe-edge-identity/edge-identity-api-reference.md#getidentities) API to retrieve all the identifier data stored locally by the SDK and send this data to your servers.
+
+When using both Edge Network extension and Adobe Solutions extensions,  use both [Identity.getIdentities](../using-mobile-extensions/adobe-edge-identity/edge-identity-api-reference.md#getidentities) API and [MobileCore.getSdkIdentities](../using-mobile-extensions/mobile-core/ile-core-api-reference.md#retrieving-stored-identifiers) API  to retrieve all the identifier data stored locally by the SDK and send this data to your servers.
 
 ## Configuration keys
 
-To update the SDK configuration, programmatically, use the following information to change your default consent values. For more information, [ConsentAPI reference](../using-mobile-extensions/adobe-edge-consent/edge-consent-api-reference.md).
+To update the SDK configuration, programmatically, use the following information to change your default consent values. For more information, see [Configuration API reference](../using-mobile-extensions/mobile-core/configuration/configuration-api-reference.md).
 
 | Key | Description |
 | :--- | :--- |
-| `consent.default` | Defines the set of default consent preferences for the SDK in XDM format. [More info.](https://github.com/adobe/xdm/blob/fc0773107f29928e1dc4753f8f055836083ea53f/docs/reference/mixins/profile/profile-consents.schema.md) |
+| `consent.default` | Defines the set of default consent preferences for the SDK in XDM format. For more details, see [Privacy/Personalization/Marketing Preferences (Consents) Schema](https://github.com/adobe/xdm/blob/fc0773107f29928e1dc4753f8f055836083ea53f/docs/reference/mixins/profile/profile-consents.schema.md). |
```

Comment:
Should this link to the `master` branch instead of a specific commit? 
https://github.com/adobe/xdm/blob/master/docs/reference/mixins/profile/profile-consents.schema.md